shell/common/rasterizer.cc

```diff
@@ -130,8 +157,13 @@ void Rasterizer::DoDraw(std::unique_ptr<flutter::LayerTree> layer_tree) {
   PersistentCache* persistent_cache = PersistentCache::GetCacheForProcess();
   persistent_cache->ResetStoredNewShaders();
 
-  if (DrawToSurface(*layer_tree) == RasterStatus::kSuccess) {
+  RasterStatus raster_status = DrawToSurface(*layer_tree);
+  raster_status = DrawToSurface(*layer_tree);
+  if (raster_status == RasterStatus::kSuccess) {
     last_layer_tree_ = std::move(layer_tree);
+  } else if (raster_status == RasterStatus::kResubmit) {
+    last_layer_tree_ = std::move(layer_tree);
```

Comment:
@chinmaygarde to confirm that the semantic change of `last_layer_tree_` from "the last layer tree that was rasterized" to the "last layer tree that attempted to be rasterized and have not yet failed or have rasterized successfully" makes sense. Actually it being non trivial to describe maybe hints that we should just add another member just for the new purpose, `resubmitted_layer_tree_` ?

Reply:
Agreed. It does not really make sense to re-purpose the current field. Will create a new member.